core/src/apps/bitcoin/common.py

```diff
@@ -212,3 +212,48 @@ def format_fee_rate(
         shortcut = ""
 
     return f"{fee_rate_formatted} sat{shortcut}/{'v' if coin.segwit else ''}B"
+
+
+def descriptor_checksum(desc: str) -> str:
```

Comment:
Can we make a comment here that the function is identical to the one found in `src/trezorlib/tools.py`, just for cross-reference.

Reply:
Good idea, fixed in [5b8865a](https://github.com/trezor/trezor-firmware/pull/3505/commits/5b8865a4a94f5bfc21c01ed6b2b6feda5ff41ea0).